rholang/src/main/scala/coop/rchain/rholang/interpreter/error_handling/package.scala

```diff
@@ -0,0 +1,25 @@
+package coop.rchain.rholang.interpreter
+
+import cats.implicits._
+import cats.MonadError
+import cats.effect.concurrent.Ref
+
+package object error_handling {
```

Comment:
Why not introduce and nest the `handling` package under `error` and avoid the ugly underscore in the name?
I would even opt for getting rid of this package object completely and putting it's contents into the `error` package object

Reply:
Just took a peek at the next commit. I find the name `error_handling.errors` confusing

Reply:
It seems reasonable to keep the error classes and the error handler separate, no?